virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc

```diff
@@ -4,17 +4,7 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-viewing-ip-of-vm-vnic
 toc::[]
 
-The QEMU guest agent runs on the virtual machine and passes the IP address of attached NICs to the host, allowing you to view the IP address from both the web console and the `oc` client.
-
-== Prerequisites
-
-. Verify that the guest agent is installed and running by entering the following command:
-+
-----
-$ systemctl status qemu-guest-agent
-----
-+
-. If the guest agent is not installed and running, xref:../../../virt/virtual_machines/vm_networking/virt-installing-qemu-guest-agent.adoc#virt-installing-qemu-guest-agent[install and run the guest agent on the virtual machine].
+You can view the IP address for a Network Interface Card (NIC) by using the web console or the `oc` client. 
```

Comment:
Examples of IP reports we show in https://bug-1876474-view-ip-address--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.html won't be available without guest agent available. I know we cannot cross reference, but IMHO it would be worth mentioning guest agent at least by name so people connect it with the guest agent section.

Reply:
Content was updated.